File: Domain/Entities/Shared/RootBase.cs

using System;
using Domain.Entities.Authentication;
using Domain.Entities.Main.Companies;

namespace Domain.Entities.Shared
{
    public abstract class RootBase
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public MyUser User { get; set; }
        public int CompanyId { get; set; }
        public Company Company { get; set; }
        public DateTime Deleted { get; set; }
        public DateTime Registered { get; set; }
    }

}

File: Views.old/src/shared/entities-dtos/root-base.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { CompanyDto } from "./company-dto";

export abstract class RootBase {
    id: number;
    userId: number;
    user: MyUser;
    companyId: number;
    company: CompanyDto;
    deleted: Date;
    registered: Date;
}